aten/src/ATen/native/transformers/cuda/attention.cu

```diff
@@ -828,6 +828,8 @@ _flash_attention_forward(
             softmax_scale,
             false /*zero_tensors*/,
             is_causal,
+            -1, /*window_size_left*/
```

Comment:
Hi, are there plans to expose `window_size_left` and `window_size_right` via sdpa? Currently having to use flash_atnn lib directly for this and does not play nicely with `torch.compile` 🥲

Reply:
Yes, there are plans to expose this functionality, however we are unlikely to expose to the top level SDPA and will expose this through the pattern described in this RFC:
https://github.com/pytorch/pytorch/issues/110681

And hopefully this will be landing soon:
https://github.com/pytorch/pytorch/pull/113103